src/app/features/event/event-card/event-card.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="event-card">
+  <div class="event-card__info">
+    <img class="info__event-img" src="assets/eventCard.png" alt="event image" />
+    <div class="info__event-details">
```

Comment:
Zdjęcie się trochę rozjeżdża (jest zbyt wysokie). Nie wiem czy to nie jest po prostu problem tego konkretnego zdjęcia. 
Trzeba zobaczyć jak to będzie wyglądało z innymi zdjęciami, bo nie jesteśmy pewni w jakiej formie my będziemy otrzymywać je z backendu.

Reply:
Poprawiłem rozdzielczość zdjęcia